src/TargetLanguage.ts

```diff
@@ -21,7 +23,9 @@ export abstract class TargetLanguage {
         if (this._options === undefined) {
             return panic(`Target language ${this.displayName} did not set its options`);
         }
-        return this._options.map(o => o.definition);
+        const definitions = this._options.map(o => o.definition);
+        definitions.push(this._sortPropertiesOption.definition);
+        return definitions;
```

Comment:
@dvdsgl This sucks, too.  I'm adding the sort option to every single target language because we don't have a nice way to pass non-language specific options.  Should we just pass a subset of the `Options` to the target language, including the `rendererOptions` there, too?

Reply:
For what it's worth, I don't think property order should be an option—it should just always match the JSON order.

Reply:
For the time being we need it for testing. We had a test that does JSON->X vs JSON->Schema->X and compares the results. Since we're using the standard JSON parser to read schema we're not preserving the order there, so I have to use a canonical order. Ideally we would preserve the order from Schema, too, but for the time being this is easier.

I'll put something in that doesn't expose the option in the UI.